docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+   
+services:
+    django:
+        build: .
+        container_name: django
+        command: ./start-django.sh
+        volumes:
+            - .:/usr/src/app/
+            - tmp:/tmp
+        ports:
+            - "8000:8000"
+        environment:
```

Comment:
I recommend the use of `.env` files for environments configurations.

Reply:
I got it!